apps/utr-backend/src/endpoints/entry/EntryRoute.ts

```diff
@@ -11,13 +11,11 @@ const EntryService = require("./EntryService");
 const EntryUtils = require("../utils/EntryUtils");
 
 router.get("/search", function (req, res, next) {
-	const latAsNumber = EntryUtils.parseToNumber(req.query.latitude);
-	const longAsNumber = EntryUtils.parseToNumber(req.query.longitude);
 	EntryService.searchEntries(
 		req.query.jobname,
-		latAsNumber,
-		longAsNumber,
-		function (error: Error | ApplicationError, result: Entry[]) {
+		req.query.latitude,
+		req.query.longitude,
+		function (error: Error | ApplicationError, result: any) {
```

Comment:
Ich musste hier den TypeScript Typ von `Entry[]` auf `any` setzen, da durch das Hinzufügen von `distance` im Service bei der Suche dort logischerweise kein `Entry[]` mehr zurückgegeben wird, sondern ein Array von Objekten.
Würde man in dem Fall manuell einen Type für `Entry`+`distance` erstellen und davon ein Array angeben?
Oder wie würde man das machen? Denn auf `any` sollte man es denke ich nicht lassen.

Reply:
Keine Ahnung, @SkyGuardian42 ?

Reply:
Theoretisch würde man im Service grundsäzlich nicht die Interfaces im Rückgabewert verwenden, die Prisma generiert, sondern eigene schreiben, um Änderungen, die nicht von Prisma unterstützt werden, einzufügen. Ist aber dann schnell eine Menge Mehraufwand. Ihr könnt für diesen einen Fall im Service theretisch einfach ein neues Interface deklarieren, das Entry extendet:
```ts
 export interface PostionedEntry extends Entry {
  longitude: number;
  latitude: number;
 }
```

Reply:
Okay verstehe, danke. Hab es jetzt mit extends Entry gemacht.